bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/RFXComBindingConstants.java

```diff
@@ -242,4 +242,6 @@
                     put(PacketType.WIND, RFXComBindingConstants.THING_TYPE_WIND);
                 }
             });
+
+    public static final String THREAD_NAME_PREFIX = "OH-" + BINDING_ID + "-";
```

Comment:
The binding id is already in in the thing uid. There make it `OH-binding-` that will make it easier to find all binding specific threads.

Reply:
Ok, I am going to change that.